README.md

```diff
@@ -641,6 +642,12 @@ $ REVIEWDOG_TOKEN="<token>" reviewdog -reporter=github-pr-check
 $ REVIEWDOG_GITHUB_API_TOKEN="<token>" reviewdog -reporter=github-pr-review
 ```
 
+## Exit codes
+By default Reviewdog will return `0` as exit code once all actions done (linter report analyzed, comments posted and so on)
+
+Once `fail-on-error` flag passed - Reviewdog will return `1` as exit code if at least one violation was found/reported.
+This can be helpful when you are using it as a step in your CI pipeline and want to mark the step failed if any error found by linter 
```

Comment:
```suggestion
This can be helpful when you are using it as a step in your CI pipeline and want to mark the step failed if any error found by linter.
```